RicoModels_pkg/ricomodels/backbones/train_backbones.py

```diff
@@ -156,29 +163,34 @@ def train_model(
     )
 
     device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
-    criterion = torch.nn.BCEWithLogitsLoss()
+    criterion = FocalLoss(use_focal_binary_multi_class=True) # torch.nn.BCEWithLogitsLoss()
```

Comment:
[Have another ticket in case this needs to be trained further](https://rico-jia.atlassian.net/browse/DL-18)